tools/explorer/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_custom_target(explorer
+  COMMENT "Building tt-explorer..."
+  COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/* ${CMAKE_CURRENT_BINARY_DIR}
+  COMMAND pip install tt_adapter
```

Comment:
Does this actually build? I was playing around with your repo and this command seems to try and query pip for a `tt_adapter` package.

Replacing it with: `pip install ${CMAKE_CURRENT_SOURCE_DIR}/tt_adapter` worked locally for me.

Reply:
Hmm, i might have changed that. The idea is to install from a local path as you did.